build/dependencies.props

```diff
@@ -32,7 +32,7 @@
     <MicrosoftCodeAnalysisVisualBasicPackageVersion>2.3.1</MicrosoftCodeAnalysisVisualBasicPackageVersion>
     <MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion>2.3.1</MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion>
     <MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>2.3.1</MicrosoftCodeAnalysisWorkspacesCommonPackageVersion>
-    <MicrosoftCSharpPackageVersion>4.5.0-preview1-25914-04</MicrosoftCSharpPackageVersion>
+    <MicrosoftCSharpPackageVersion>4.5.0-preview1-26006-06</MicrosoftCSharpPackageVersion>
```

Comment:
Can you check if these binaries are strong name signed?

Reply:
![capture](https://user-images.githubusercontent.com/7574801/33787231-3501b76c-dc21-11e7-9955-5f7dc2cd69ad.JPG)